packages/network-controller/src/NetworkController.test.ts

```diff
@@ -1396,140 +779,6 @@ describe('NetworkController', () => {
           expect(fakeMetamaskProviders[0].stop).toHaveBeenCalled();
         });
       });
-
-      it('records the version of the current network in state (assuming that the request for net_version is made successfully)', async () => {
```

Comment:
I think we should keep this test, but we can tweak the name and change the expectation.